addons/bestja_detailed_reports/models.py

```diff
@@ -13,6 +13,11 @@ class Project(models.Model):
         compute_sudo=True,
         search='_search_use_detailed_reports',
     )
+    stores_in_project = fields.One2many(
```

Comment:
This field is not specific to detailed reports and should be added in the `bestja_stores` modules. In fact it will be added there when my pull request #206 comes through.